src/scalar.rs

```diff
@@ -401,6 +411,7 @@ use serde::de::Visitor;
 use serde::{self, Deserialize, Deserializer, Serialize, Serializer};
 
 #[cfg(feature = "serde")]
+#[cfg_attr(docsrs, doc(cfg(feature = "serde")))]
```

Comment:
If you add `#![cfg_attr(docsrs, feature(doc_auto_cfg))]` to the crate attributes, `doc(cfg)`s that exactly mirror regular `cfg`s at the same level will be unnecessary.

Reply:
Unfortunately it leaves an "or `docsrs`" note on all the things in `backend`. So I did it manually 🤷‍♂️

Reply:
Hmm, maybe I can do `doc_auto_cfg` and then just do manual things for the `backend` parts

Reply:
> Unfortunately it leaves an "or `docsrs`" note on all the things in `backend`. So I did it manually :man_shrugging:

I think that should be fixable through `#![cfg_attr(docsrs, feature(doc_cfg_hide))]` + `#![doc(cfg_hide(docsrs))]`.

Reply:
Fixed. Thanks for the idea!

Reply:
I don't actually see where you enabled it in the commit that says you did, but happy to help 😄